### ISSUE TITLE
Load PDB Failed with NullPointerException

### ISSUE THREAD

**Issue by mcfi** [NationalSecurityAgency/ghidra#138]
**Describe the bug**

Load PDB File fails with NullPointerException thrown when loading certain xml files as PDBs.

**To Reproduce**

1. Save the following xml file locally.

```
<pdb file="C:\Users\whatever\source\repos\seh\x64\Debug\seh.pdb" exe="seh" guid="{C4DCE96F-27AB-4189-9A6C-D542A986A69E}" age="1">
    <tables>
        <table name="SourceFiles">
            <source_file name="C:\Users\whatever\AppData\Local\Temp\lnk{1F6DAF19-31D5-4B81-B0A7-A061310EA8F9}.tmp" id="0x2" /> 
        </table>
        <table name="LineNumbers">
        </table>
        <table name="Sections">
        </table>
        <table name="SegmentMap">
            <segment number="1" address="0x1000" />  
        </table>
        <table name="InjectedSource">
        </table>
        <table name="FrameData">
        </table>
        <table name="InputAssemblyFiles">
        </table>
    </tables>
</pdb>
```

2. Open CodeBrowser.

3. Click "File", and then "Load PDB File", select the xml file.

4. A window will pop up and display the error of  "Problem parsing or applying PDB information: NullPointerException"

**Expected behavior**
No NullPointerException should be thrown.

**Screenshots**

![image](https://user-images.githubusercontent.com/6384579/54073853-beec2d00-4240-11e9-9770-4a99994c096d.png)

**Environment (please complete the following information):**
 - OS: Windows 10
 - Java Version: 11
 - Ghidra Version: 9.0

**Comment by kant2002** [comment]
FYI. There duplicate report by me #188  (my bad to duplicate) where I find some simple quick fix.

**Comment by ghidra1** [comment]
This will be corrected for 9.0.1.

Newer PDBs omit unused sections. A work around is to add the missing XML sections to the top of the PDB xml file until 9.0.1 is available. Do not add if corresponding section already exists in file.

```
<pdb ....>
     <classes/>
     <functions/>
     <datatypes/>

...snip...

</pdb>

```

**Comment by ghidra1** [comment]
This will be corrected in our next release  9.0.1
